release/tag_repos.sh

```diff
@@ -194,6 +206,10 @@ case "${subcmd}" in
 status)
 	check_versions
 	;;
+pre-release)
+	runtime_repo="github.com/kata-containers/runtime"
+	check_versions $(cat "${GOPATH}/src/${runtime_repo}/VERSION")
```

Comment:
Almost there, I wonder if instead of assume that the runtime repo exist  here, could you read the version as a second parameter?
so in travis you can do something like 
`tag_repos.sh  pre-release $(cat VERSION) `
this also helps to remove the GOPATH variable and runtime_repo
- Also could you add some usage info ? 
- finally you can add some basic test here tag_repos_test.sh
`tag_repos.sh  pre-release 100000` will fail 
or
`tag_repos.sh  pre-release $(curl https://github.com/kata-containers/runtime/blob/master/VERSION)` I think should work